### PR TITLE
Mobile menu review

### PR DESCRIPTION
**Pull request by mprotic123**
This pull request is created in order to review the project "Portfolio-mobile menu" from Microverse.

### CODE REVIEW

File: personal-portfolio/index.html [hunk 1]
```diff
@@ -4,356 +4,442 @@
...
-          <i class="material-icons">menu</i>
-        </button>
+    <header>
+      <nav>
+        <h3><a href="#">Welcome</a></h3>
+        <ul>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
```
Comment (rloterh) [inline personal-portfolio/index.html:37]: - [ ] Your mobile menu does not disappear when any of the menu options: `Portfolio`, `About` or `Contact` is clicked or tapped. To enhance the UX (User Experience) we actually the mobile menu to vanish and jump to the selected content when a user taps or clicks on the mobile menu option: 

**Your mobile Screen (When mobile _Portfolio_ option is tapped):**
image

**Your mobile Screen (When mobile _About_ option is tapped):**
image

**Your mobile Screen (When mobile _Contact_ option is tapped):**
image

Kindly ensure to improve upon the issues listed above to enhance a better UX.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Milan and Dannie, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- Mobile UI design matches given template design
- No linter errors


## Required Changes ♻️
- [ ] _Check the comments under the review._


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: personal-portfolio/js/script.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const menu_btn = document.querySelector('.hamburger');
+const mobile_menu = document.querySelector('.mobile-nav');
+
+menu_btn.addEventListener('click', function () {
+  menu_btn.classList.toggle('is-active');
+  mobile_menu.classList.toggle('is-active');
+});
```
Comment (dasileker) [inline personal-portfolio/js/script.js:7]: - Hello nice job on fixing the errors mentioned by the previous TSE, but you are still on an issue.

- The close button does not disappear after clicking on it to close the menu we should see the hamburger icon instead
porfolio

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @mprotic123** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hemantso — APPROVED** [review]
Hi @mprotic123, 

Great work on making the requested changes. 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**